text-to-speech/versta/export/__main__.py

```diff
@@ -97,18 +126,23 @@ def main(
     # Step 4: Save the tokenizer files
     tokenizer_files = save_tokenizer(converted_dir, output_dir)
 
-    # Step 5: Get all voices from voices directory
-    voices = get_voices(output_dir / "voices")
+    # Step 5: Get all voices from voices directory (for Kokoro) or handle Piper voices
+    if model_format == "kokoro":
+        voices = get_voices(output_dir / "voices")
+    else:
+        # Piper models don't use separate voice files like Kokoro
+        voices = []
```

Comment:
Move this inside the `get_voices` method instead so that there are not that many if statmeents in `__main__.py`